Source/WebCore/page/NavigatorLoginStatus.h

```diff
@@ -26,31 +26,34 @@
 #pragma once
 
 #include "Supplementable.h"
+#include <wtf/WeakPtr.h>
 
 namespace WebCore {
 
 class DeferredPromise;
 class Navigator;
-class NavigatorLoginStatus final : public Supplement<Navigator> {
+class WeakPtrImplWithEventTargetData;
```

Comment:
This should be removed.

Reply:
Why? We need to extend Navigator, no?

Reply:
you mean remove the extending navigator?

Reply:
The forward declaration should be removed (`class WeakPtrImplWithEventTargetData;`)